subsys/net/lib/http/Kconfig

```diff
@@ -219,6 +219,16 @@ config HTTP_SERVER_COMPRESSION
 	    5. deflate  -> .zz
 	    6. File without compression
 
+config HTTP_SERVER_STATIC_FS_RESPONSE_SIZE
+	int "Size of static file system response buffer"
+	depends on FILE_SYSTEM
+	default 1024
```

Comment:
No stack overflow with such a large buffer? I wonder if it's needed to adjust the defaults for the server's stack size.

Reply:
I think that it is indeed too small.
My current setting (and it's enough for sending things in 2KB chunks in both direction and serving ca. 1MB files) is
 `CONFIG_HTTP_SERVER_STACK_SIZE=8192`
 
 I'd probably suggest a default of at least 4096 (now it is 3072)

Reply:
We could have a conditional default `4096 if FILE_SYSTEM` (just make sure it's first before the unconditional one). Current default is fine I think for now (i've seen no issues), so if we increase the stack by 1KB, I think we could have the buffer set to 1024.